src/components/ShowWinner/ShowWinner.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components/macro'
+
+export default function ShowWinner({ visible, title, players, confetti }) {
+  const winners = players.sort((a, b) => a.score - b.score)
+  const newWinners = []
```

Comment:
You could use `.filter()` to get a new array if the winner score is equal to the winners. I think.
```suggestion
  const newWinners = winners.filter(winner => winner.score === winners[0].score) 
```